docs/src/mdx-components.tsx

```diff
@@ -17,7 +19,22 @@ interface MDXComponents {
 }
 
 export const mdxComponents: MDXComponents = {
-  a: (props) => <Link {...props} />,
+  a: (props) => {
+    if (props.href.startsWith('http')) {
+      return (
+        <Link
+          target="_blank"
+          rel="noopener"
+          {...props}
+          className={clsx(props.className, 'inline-flex items-center gap-1')}
+        >
+          {props.children}
+          <ExternalLinkIcon />
+        </Link>
+      );
+    }
```

Comment:
Should we put it in the actual `Link` component so it can be reused in JSX too? (We'd need to clean the icon from the 404 page then).